modules/capi-creating-infrastructure-resource.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cluster_api_machine_management/cluster-api-configuration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="capi-creating-infrastructure-resource_{context}"]
+= Creating a Cluster API infrastructure resource
+
+You can create a provider-specific infrastructure resource by creating a YAML manifest file and applying it with the {oc-first}.
+
+.Prerequisites
+
+* You have deployed an {product-title} cluster.
+
+* You have enabled the use of the Cluster API.
+
+* You have access to the cluster using an account with `cluster-admin` permissions.
+
+* You have installed the {oc-first}.
+
+* You have obtained the cluster ID value.
+
+* You have created and applied the cluster resource.
+
+.Procedure
+
+. Create a `<infrastructure_resource_file>.yaml` file similar to the following:
++
+--
+[source,yaml]
+----
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: <infrastructure_kind> # <1>
+metadata:
+  name: <cluster_name> # <2>
+  namespace: openshift-cluster-api
+spec: # <3>
+----
+<1> Specify the infrastructure kind for the cluster. This value must match the value for your platform. The following values are valid:
+* `AWSCluster`: The cluster is running on {aws-first}.
+* `GCPCluster`: The cluster is running on {gcp-first}.
+<2> Specify the name of the cluster.
+<3> Specify the details for your environment. These parameters are provider specific. For more information, see the sample Cluster API infrastructure resource YAML for your provider.
+--
+
+. Create the infrastructure CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f <infrastructure_resource_file>.yaml
+----
+
+.Verification
+
+* Confirm that the infrastructure CR is created by running the following command:
++
+[source,terminal]
+----
+$ oc get <infrastructure_kind>
+----
++
+where `<infrastructure_kind>` is the value that corresponds to your platform.
++
+.Example output
```

Comment:
The example output is for aws, for gcp it looks like below 
```
oc get gcpcluster
NAME                 CLUSTER              READY   NETWORK
huliu-gcp13a-vnh8b   huliu-gcp13a-vnh8b   true    huliu-gcp13a-vnh8b-network
```

Reply:
Thanks! I am thinking that probably the headings `NAME`, `CLUSTER`, and `READY` will be present for all platforms, right? I can just trim this an make a note that other headings are provider specific in that case.